CoreLibs/src/org/sleuthkit/autopsy/corelibs/OpenCvLoader.java

```diff
@@ -57,4 +67,13 @@ public static boolean isOpenCvLoaded() throws UnsatisfiedLinkError {
         }
         return OPEN_CV_LOADED;
     }
+    
+    /**
+     * Return whether OpenCV library has been loaded.
+     * 
+     * @return true if OpenCV library was loaded, false if not.
+     */
+    public static boolean hasOpenCvLoaded() {
+        return OPEN_CV_LOADED;
```

Comment:
I know that the OPEN_CV_LOADED identifier is not new, but while we are in here we should put it in standard varibale name camel case, since this is not a constant.